xen/xenops_types.ml

```diff
@@ -39,6 +40,7 @@ module Vgpu = struct
     virtual_pci_address: address option;
     type_id: string;
     uuid: string;
+    extra_args: string;
```

Comment:
If we expect this to be key/value pairs, this should not be a string but an association list as we otherwise end up with quoting problems. For example, a value better now does not contain `=`. Has this been thought trough?

Reply:
yes, That is my first throught. 
But when I come to how it currently works, it is a string in the VM platform, (https://info.citrite.net/display/engp/vGPU+productisation+debug+API),
From this point of view, it maybe a good idea to keep the user experence consistency. 
By the way, this is only for Invida debug, we just pass whatever set to demu without any process.

Reply:
I agree that if we pass on a string, this is correct. Maybe update the code comment:

```
    extra_args: "key1=v1,key2=v2,key3=v3" - string is passed on as is and no structure is assumed
```

Reply:
updated.